server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestClusterHealthAction.java

```diff
@@ -81,9 +88,15 @@ public static ClusterHealthRequest fromRequest(final RestRequest request) {
         if (request.param("wait_for_events") != null) {
             clusterHealthRequest.waitForEvents(Priority.valueOf(request.param("wait_for_events").toUpperCase(Locale.ROOT)));
         }
-        clusterHealthRequest.return200ForClusterHealthTimeout(
-            request.paramAsBoolean("return_200_for_cluster_health_timeout", clusterHealthRequest.doesReturn200ForClusterHealthTimeout())
-        );
+        String return200ForClusterHealthTimeout = request.param("return_200_for_cluster_health_timeout");
```

Comment:
Please use a named constant for the param name both here and in the deprecation message to make it clear it's not just a coincidence that they're the same string. It's ok to use the literal string in tests.